### ISSUE TITLE
Add some card/tab with logs on jupyter web app like lnx terminal

### ISSUE THREAD

**Issue by ManPython** [jupyterlab/team-compass#171]
### Problem

Whe jupyter is used on hosting, cloud when user is no admin or if app is run as screen comand via terminal, then hard to analyse some problems/issues, logs.

### Proposed Solution

Want suggest to add some card/tab with logs on jupyter web app like lnx terminal.

**Comment by fcollonval** [comment]
This repository does not seems to fit your issue. You are speaking of server logs for JupyterLab?

One comment on the issue itself, the way logs are stored/configured is highly dependent on the hosting platform set up. So having a generic way may not be easy.

**Comment by ManPython** [comment]
> You are speaking of server logs for JupyterLab?

Yes, as Python. Sure.. are stored, but must operate on terminal, restart app outside screen function to see whats is wrong, wg. why reseting, why binding on series of python versions not working, how many resets, etc.
I was in idea that this can be good option to offer for some users this tatb, eg. configured privilages in configs to access this.
This touching both JupyterLab and pure Jupyter. Wrong addres?
